iOS/Flows/Home/_New/new_ConversationViewController.swift

```diff
@@ -150,36 +191,68 @@ extension new_ConversationViewController {
                                     collectionView: self.collectionView,
                                     animationCycle: animationCycle)
     }
-}
 
-extension new_ConversationViewController: ChatChannelControllerDelegate {
+    func subscribeToConversationUpdates() {
+        self.conversationController.messagesChangesPublisher.mainSink { [unowned self] changes in
+            Task {
+                await self.updateMessages(with: changes)
+            }
+        }.store(in: &self.cancellables)
```

Comment:
Using the publisher instead of the delegate to listen for message updates.

Reply:
sick